css/style.css

```diff
@@ -0,0 +1,173 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  box-sizing: border-box;
+}
+
+.profile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 40px;
+}
+
+.profile-picture {
+  width: 150px;
```

Comment:
I don't see any changes if we remove this code.

By the way, if you want to specify size for some image, do it with HTML:
```
<img width="150" height="150" />
```
https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/